### DISCUSSION TITLE
Showcase: fontkit doc site https://fontkit.typogram.co/

### DISCUSSION THREAD

**Discussion by Typogram** [GitbookIO/community#322] [Show and Tell]
[Fontkit Documentation Site](https://fontkit.typogram.co/)
[Fontkit Github](https://github.com/foliojs/fontkit)
Here’s what sets it apart:
- Depth Over Breadth: While the official documentation of fontkit is limited to a README page on its git repo, I delved deeper. I meticulously explored each object, instantiated them in a live code environment, logged all their properties and methods (including private ones), and crafted a thorough API reference.
- Searchable Documentation: Built using GitBooks, my documentation ensures that users can easily search for and find the information they need. The intuitive search functionality streamlines the process, making information access quick and seamless.
- AI-Enhanced Interactions: incorporat AI into the doc site. Now, users can pose questions, and the AI will craft responses based on the content of the documentation. This feature promises real-time assistance and an enriched user experience.
![request-name-change-4](https://github.com/GitbookIO/community/assets/41246474/7ca1477f-f22d-44af-8a11-0a54de64a7f4)

**Comment by addisonschultz** [comment]
This is very nice! And love that you're using Lens for AI-enhanced interactions. Great overview of the structure of your site too 👌

**Reply by Typogram** [reply]
Thanks @addisonschultz I have just submitted my site for https://www.gitbook.com/showcase via the Google form. Any change of getting added to there?

**Reply by addisonschultz** [reply]
Hi @Typogram! Thank you for your submission 😄 

However, we've recently made the decision to close submissions for the GitBook Library, in light of a new directory of GitBook sites that is auto curated - one that you can find at https://www.gitbook.com/open/library/

We would love to continue exploring the library in the future, but for the time being the project is no longer running. Thank you for your contribution though!

**Reply by Typogram** [reply]
ok, thanks for letting me know.
you should disable the submission link on the showcase site to avoid waste of time for the future.
